### PR TITLE
Added getinfo

### PR DESCRIPTION
**Pull request by benzcash**
Signed-off-by: Ben Wilson <ben@z.cash>

- get the network name the node is connected to
- get the nodes version

### CODE REVIEW

**Comment by charlieok** [comment]
'doubtingben'? Is there a story there? :)

File: collector.go [hunk 1]
```diff
@@ -6,6 +6,18 @@ import (
 
 //Define the metrics we wish to expose
 var (
+	zcashdBlockchainInfo = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "zcash_network",
```
Comment (charlieok) [inline collector.go]: looks like we're breaking from a naming pattern here wherein these 'Name' attributes closely follow the go variable names they're assigned to. In this case 'network (name)' is one of the two components (the other being '(number of) blocks'). Is "zcash_network" a better choice of 'Name' attribute here than something like "zcash_blockchain_info"?

**Comment by benzcash** [comment]
> 'doubtingben'? Is there a story there? :)

My personal account, working off hours :grin:

**Review by charlieok — APPROVED** [review]
ACK

**Pull request merged**